frontend/packages/ux-editor/src/components/config/componentSpecificContent/FileUpload/FileUploadComponent.tsx

```diff
@@ -70,21 +70,20 @@ export const FileUploadComponent = ({
         propertyPath={`${component.propertyPath}/properties/hasCustomFileEndings`}
       >
         {({ value }) => (
-          <LegacyRadioGroup
-            items={[
-              {
-                label: t('ux_editor.modal_properties_valid_file_endings_all'),
-                value: 'false',
-              },
-              {
-                label: t('ux_editor.modal_properties_valid_file_endings_custom'),
-                value: 'true',
-              },
-            ]}
+          <Radio.Group
             name={`${component.id}-valid-file-endings`}
-            variant='horizontal'
+            inline={true}
             value={value === true ? 'true' : 'false'}
-          />
+          >
+            {Children.toArray([
```

Comment:
as mentioned earlier in this review, I think we can avoid using the Children.toArray property. 😄